### PR TITLE
chore(ts): Convert onboardingStatus

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Also converts less to styled components

### CODE REVIEW

**Review by billyvg — APPROVED** [review]
Doesn't look like we have any percy snapshots of this :/ or if we do, then it didn't change so 👍

File: src/sentry/static/sentry/app/components/sidebar/onboardingStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+import TodoList from 'app/components/onboardingWizard/todoList';
+import Tooltip from 'app/components/tooltip';
+import {Organization} from 'app/types';
+
+type Props = {
+  org: Organization;
+  currentPanel: string;
+  onShowPanel: () => void;
+  hidePanel: () => void;
+  showPanel: boolean;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/sidebar/onboardingStatus.tsx]: wouldn't it be better to make it optional?

**Review by priscilawebdev — COMMENTED** [review]
.

**Review by priscilawebdev — APPROVED** [review]
Great job! 👍

**Comment by EvanPurkhiser** [comment]
It should look exactly the same, so there shouldn't be any visual regressions. But you might also be right that there aren't any snapshots with it on.

**Pull request merged**